content/docs/self-hosted-runners.md

```diff
@@ -361,6 +361,39 @@ for obtaining these keys.
 ☝️ **Note** The same credentials can also be used for
 [configuring cloud storage](/doc/cml-with-dvc#cloud-storage-provider-credentials).
 
+The following are the minimum IAM permissions needed for the CML runner to
+deploy on EC2:
```

Comment:
- how about linking to https://github.com/iterative/terraform-provider-iterative/blob/master/docs/guides/permissions/aws/main.tf (just like https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/authentication#amazon-web-services does)?
- also seems to be the lists are slightly different?

/CC @0x2b3bfa0

Reply:
The lists you mention are valid for both “TPI” and `cml runner`

Reply:
That's why they're slightly different. I.e. have more permissions.

Reply:
`cml runner` does require less, not using the autoscaling groups, etc.

Reply:
Should I add a link or no? If the terraform provider iterative is always up-to-date, it seems better to link to it rather than have the same permissions listed twice, even if there are extras :shrug: